packages/replay/README.md

```diff
@@ -1,4 +1,4 @@
-# sentry-replay
+# @sentry/replay
 
 Note: Session Replay is currently in beta.
 
```

Comment:
This comment is meant for line 7:

Since we changed stuff recently in the SDK, should we recommend 7.23.0 as a minimum version? Also, I'd add here that users should always use the same Replay version as the SDK version.

Reply:
good point, will mention this!

Reply:
Added a paragraph:


> Make sure to use the exact same version of `@sentry/replay` as your other Sentry package(s), e.g. `@sentry/browser` or `@sentry/react`.